lambda/script.py

```diff
@@ -0,0 +1,154 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+import boto3  # type: ignore
+import googleapiclient.discovery  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+
+def get_env_var(env) -> Optional[str]:
+    """
+    Returns an enviroment variable.
+    """
+    return os.environ.get(env)
```

Comment:
what does this function add?

Reply:
This function has been removed in favour of using os.environ directly.